clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1419,6 +1419,35 @@ is not specified.
   }];
 }
 
+def ExplicitInitDocs : Documentation {
```

Comment:
This is a little lacking for me?  Semantics, limitations, examples of how this is intended to be used, warnings about 'make sure you own the type' sorta thing would be appreciated.